wishlists/views.py

```diff
@@ -34,4 +34,15 @@ def patch(self, request):
                 return JsonResponse({"Result": result}, status=206)
 
         except Content.DoesNotExist:
-            return JsonResponse({"Result": "CONTENT_DOES_NOT_EXIST"}, status=404)
\ No newline at end of file
+            return JsonResponse({"Result": "CONTENT_DOES_NOT_EXIST"}, status=404)
+
+    @token_validation_decorator
+    def get(self, request):
+        wishlists =  Wishlist.objects.filter(user=request.user, like = True).select_related('user', 'content')
+
+        result  = [{
+            "user_id"    : wishlist.user.id,
+            "content_id" : wishlist.content.id,
+        } for wishlist in wishlists]
```

Comment:
wishList column에 이미 user_id를 가지고 있는데 `wishlist.user.id`를 통해서 user_id를 가져오는 것은 불필요한 엑션이 추가되는 것 같습니다. 그 여파로 user table을 JOIN해서 가져와야되는 문제가 발생하네요.

```suggestion
        wishlists =  Wishlist.objects.filter(user=request.user, like = True).select_related('user', 'content')

        result  = [{
            "user_id"    : wishlist.user.id,
            "content_id" : wishlist.content.id,
        } for wishlist in wishlists]
```